docker-compose.yaml

```diff
@@ -8,3 +8,10 @@ services:
     environment:
       APPINSIGHTS_CLOUDROLE: fcp-fd-comms-local
       APPINSIGHTS_CONNECTIONSTRING: ${APPINSIGHTS_CONNECTIONSTRING}
+      MESSAGE_QUEUE_HOST: ${MESSAGE_QUEUE_HOST}
+      MESSAGE_QUEUE_USER: ${MESSAGE_QUEUE_USER}
+      MESSAGE_QUEUE_PASSWORD: ${MESSAGE_QUEUE_PASSWORD}
+      MESSAGES_TOPIC_ADDRESS: fcp-fd-comms-${DEV_SUFFIX}
+      MESSAGES_SUBSCRIPTION_ADDRESS: fcp-fd-comms-${DEV_SUFFIX}
```

Comment:
What if we renamed these `MESSAGE_QUEUE_SUFFIX` then they align to all other repos in the programme and devs won't need to have both?

Reply:
Done.